diff --git a/debian/rules b/debian/rules
index 9ec22bc..46c4c33 100755
--- a/debian/rules
+++ b/debian/rules
@@ -23,6 +23,13 @@ else ifeq ($(DEB_HOST_ARCH),aarch64)
 		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
 		-DARM_DYNAREC=1 \
 		-DNOGIT=1
+else ifeq ($(DEB_HOST_ARCH),loong64)
+	dh_auto_configure -- \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DLARCH64=1 \
+		-DLARCH64_DYNAREC=1 \
+		-DBOX32=ON -DBOX32_BINFMT=ON \
+		-DNOGIT=1
 else
 	dh_auto_configure -- \
 		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
